src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/mgr-modules/mgr-module-list/mgr-module-list.component.spec.ts

```diff
@@ -147,55 +141,11 @@ describe('MgrModuleListComponent', () => {
         always_on: false
       });
       component.updateModuleState();
-      tick(2000);
-      tick(2000);
+      tick(mgrModuleService.REFRESH_INTERVAL);
+      tick(mgrModuleService.REFRESH_INTERVAL);
       expect(mgrModuleService.enable).toHaveBeenCalledWith('foo');
       expect(mgrModuleService.list).toHaveBeenCalledTimes(2);
-      expect(notificationService.suspendToasties).toHaveBeenCalledTimes(2);
-      expect(component.blockUI.start).toHaveBeenCalled();
-      expect(component.blockUI.stop).toHaveBeenCalled();
       expect(component.table.refreshBtn).toHaveBeenCalled();
     }));
-
-    it('should disable module', fakeAsync(() => {
```

Comment:
I see you moved to the logic to mgrmodule component. I think we should also move the tests or adapt current ones if required. PTAL

Thanks

Reply:
I moved the `updateModuleState` function from the `mgr-module-list.component` to the service `mgr-module.service` as it seemed more appropiate and easier to reuse the code.
I though I had properly moved the relevants tests already, I'll recheck as I see the make check failing